website/docs/r/dashboard.html.markdown

```diff
@@ -683,6 +709,10 @@ Nested `widget` blocks have the following structure:
             - `log_query`: (Optional) The log query to use in the widget. The structure of this block is described [below](dashboard.html#nested-apm_query-and-log_query-blocks).
             - `process_query`: (Optional) The process query to use in the widget. The structure of this block is described [below](dashboard.html#nested-process_query-blocks).
             - `display_type` - (Optional) Type of display to use for the request. Available values are: `area`, `bars`, or `line`.
+            - `style` - (Optional) Style of the widget graph. One nested block is allowed with the following structure:
+              - `palette` - (Optional) Color palette to apply to the widget.
```

Comment:
Do we have docs about what values these can be? Can we link to those here?

Reply:
Good catch. For some reason markdown link syntax was causing the URLs to disappear from the docs when added in a nested list item so I just added the link URL as text instead.